### PR TITLE
Archive Changes

### PR DESCRIPTION
**Pull request by ParnnVuhe**
**Archive**
- Replaces the floors.
- Adds a more comprehensive Security System
- Adds a room in the Restricted Archive.
- Does a little cosmetic touchup.
- Adds a little quality of life changes.

Screenshot: https://gyazo.com/13964a8cedacf941980e1d2bf96b97af

### CODE REVIEW

File: html/changelogs/pvlog.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+author: ParnnVuhe
```
Comment (JakeDaBoss) [inline html/changelogs/pvlog.yml]: Should probably be DoctorOctopus. In-game CKey if you could.

**Review by JakeDaBoss — CHANGES_REQUESTED** [review]
Awaiting inspection.

**Comment by ParnnVuhe** [comment]
Closing.. Something pushed the map into a conflict in the two days since its been up.  😐

**Comment by JakeDaBoss** [comment]
Please present the changes to me in a DM, and the affected map file to make integration into the update smoother.

**Comment by JakeDaBoss** [comment]
These changes have been reviewed and I don't find them adding much, rather only drawing from the aesthetic currently present.

**Pull request closed without merging**